### PR TITLE
Add support for mongo backups

### PR DESCRIPTION
**Pull request by codejamninja**
Fixes issue #268

### CODE REVIEW

File: providers-config.default.toml [hunk 1]
```diff
@@ -20,3 +20,9 @@
 	pre_cmd = "mkdir -p $volume/backups && slapcat > $volume/backups/all.ldif"
 	detect_cmd = "[[ -f $volume/DB_CONFIG ]]"
 	backup_dir = "backups"
+	
+	[providers.mongo]
+	pre_cmd = "mongodump -o $volume/backups"
```
Comment (mcanevet) [inline providers-config.default.toml]: Shouldn't you create the `backups` directory first, like what is done on other providers?

Comment (codejamninja) [reply providers-config.default.toml]: The ‘mongodump’ command creates the backup directory.

Comment (mcanevet) [reply providers-config.default.toml]: OK. Then it's OK for me to merge this. @cryptobioz @raphink any objection?

Comment (raphink) [reply providers-config.default.toml]: LGTM

**Comment by codejamninja** [comment]
Thanks!

**Pull request merged**